_layouts/default.html

```diff
@@ -8,6 +8,7 @@
         <title>{{ site.name }}{% if page.title %} - {{ page.title }}{% endif %}</title>
         <link rel="shortcut icon" href="/images/shield-small.png">
         <meta name="description" content="{% if page.description %}{{ page.description }}{% else %}{{ site.description }}{% endif %}">
+        <link rel="author" href="humans.txt">
```

Comment:
Move up one line and change to this (with front `/`):
```html
<link rel="author" href="/humans.txt">
```

Reply:
Reordered and fixed (thought I had done that but apparently not) in 566d0c8